sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.applicationChangeRequest.e2e-spec.ts

```diff
@@ -97,6 +98,11 @@ describe("ApplicationStudentsController(e2e)-applicationChangeRequest", () => {
         applicationStatus: ApplicationStatus.Completed,
       },
     );
+    // Ensure the application data has program persistent properties.
```

Comment:
Is there any reason not to use the `saveFakeApplication` options as below to save the `applicationData`?
```ts
const completedApplication = await saveFakeApplication(
  db.dataSource,
  { student, programYear },
  {
    applicationStatus: ApplicationStatus.Completed,
    // Ensure the application data has program persistent properties.
    applicationData: {
      programPersistentProperties: ["selectedLocation"],
    } as ApplicationData,
  },
);
```